osdep/semaphore_osx.c

```diff
@@ -77,18 +78,20 @@ int mp_sem_timedwait(mp_sem_t *sem, const struct timespec *abs_timeout)
         if (abs_timeout) {
             timeout_ms = 0;
 
-            // OSX does not provide clock_gettime() either.
-            struct timeval tv;
-            gettimeofday(&tv, NULL);
+            struct timespec ts;
```

Comment:
I don't know about that... I made two related commits to change it to mp_time. See:
https://github.com/kasper93/mpv/commit/31f2f660400f3a2470be67d3862ff852a9bcd264
https://github.com/kasper93/mpv/commit/f79d6cefd5f8ad5026bbbfd9592c660cd178578c

Reply:
Well if you're rewriting all this anyway, I can just drop this commit. Not like we'll die from this `gettimeofday`.